AmazonChimeSDK/AmazonChimeSDK/audiovideo/AudioVideoControllerFacade.swift

```diff
@@ -45,6 +45,17 @@ import Foundation
     /// - Throws: `PermissionError.videoPermissionError` if video permission of `AVCaptureDevice` is not granted
     func startLocalVideo() throws
 
+    /// Start local video with configurations and begin transmitting frames from an internally held `DefaultCameraCaptureSource`.
+    /// `stopLocalVideo` will stop the internal capture source if being used.
+    ///
+    /// Calling this after passing in a custom `VideoSource` will replace it with the internal capture source.
+    ///
+    /// This function will only have effect if `start` has already been called
+    ///
+    /// Parameter config: configurations of emitted video stream, e.g. simulcast
+    /// - Throws: `PermissionError.videoPermissionError` if video permission of `AVCaptureDevice` is not granted
+    func startLocalVideo(config: LocalVideoConfiguration) throws
```

Comment:
Do we really need this? I think its fine to expect builders to write the few lines needed to inject a custom source. Long term i don't think we want a version that internally creates a source.

Reply:
We also need this for maxbitrate, right?

Reply:
Our current internal SDK API is designed as `XXXLocalVideo(source, config)` (i.e. it doesn't support calling without a source. Maybe its reasonable to have that create an internal source to make it easy for onboarding, but it requires the audio video controller to couple to the device controller logic. Anyways its not a big deal here i guess, just one extra function.